lab1/print_style.css

```diff
@@ -0,0 +1,83 @@
+body {
+  padding: 50px 100px;
+  background-color: white;
+  font-family: "Comic Sans MS", arial;
+  font-size: 100%;
+  text-align: justify;
+  text-justify: inter-word;
+}
+nav {
+  display:none;
+}
+
+a {
+  text-align: center;
+}
+
+a:link {
+    color: #000000;
+}
+
+a:visited {
+    color: #000000;
+}
+
+a:hover {
+    color: #FF00FF;
+}
+
+a:active {
+    color: #0000FF;
+}
+caption {
+  margin-bottom: 15px;
+  font-size: 20px;
+  font-weight: 700;
+  font-style: italic;
+}
+
+img {
+  display: block;
```

Comment:
Is this needed? Else maybe add a `div` around the image rather than changing the behaviour of all images in such a way.